### PR TITLE
Feature/create and edit folder

### PR DESCRIPTION
**Pull request by hannyle**
### Description

- Implementing a new modal for creating and editing folder based on new UI design: https://www.figma.com/proto/N0Y3N0nfyMS2wG3gjwZ201/SD-Connect-Prototype?node-id=526%3A5506&scaling=min-zoom&page-id=517%3A2532&starting-point-node-id=517%3A2533&show-proto-sidebar=1
- Adding possibility to view project members in MyCSC.

### Related issues
Closes https://github.com/CSCfi/swift-browser-ui/issues/612, https://github.com/CSCfi/swift-browser-ui/issues/613

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Changes Made

- Removed routes to **add** and **edit** folder
- Added modal from `csc-ui` to create and edit folder
- Added link to MyCSC to see project members
- Updated e2e test
- Modified `path` in `e2etests.yml` file to show screenshots and videos when the tests fail in github

### Testing

- [x] Integration Tests

### Mentions
- I would need help with Finnish translation for new terms used.

### CODE REVIEW

**Comment by saulipurhonen** [comment]
Maybe rebase with `devel` branch so it would be a bit easier to review?

**Review by saulipurhonen — APPROVED** [review]
Looks good to me! Functionality works as described and code formatting has improved from previous version.

**Comment by saulipurhonen** [comment]
One thing I noticed is that focus isn't trapped inside modal so user is able to navigate behind the modal with keyboard. This however seems to be problem within the CSC DS component library.

File: swift_browser_ui_frontend/src/common/lang.js [hunk 1]
```diff
@@ -526,12 +553,15 @@ let default_translations = {
       copyfail: "Säiliön kopiointi epäonnistui",
       renderFolders: "Näytä kansioina",
       tagName: "Tägit",
-      tagMessage: "Paina 'enter' lisätäksesi.",
+      tagPlaceholder: "# Add a tag and press enter",
```
Comment (sampsapenna) [inline swift_browser_ui_frontend/src/common/lang.js:556]: Translation: *Lisää tunniste ja paina rivinvaihtoa*

File: swift_browser_ui_frontend/src/common/lang.js [hunk 2]
```diff
@@ -526,12 +553,15 @@ let default_translations = {
...
       container_ops: {
         addContainer: "Luo uusi säiliö",
         editContainer: "Muokataan säiliötä: ",
-        norename: "Säiliötä ei voi nimetä uudelleen, " +
-        "mutta sen voi kopioida uudella nimellä.",
+        norename:
+          "Säiliötä ei voi nimetä uudelleen, " +
+          "mutta sen voi kopioida uudella nimellä.",
+        createdFolder: "",
+        myResearchProject: "My research project.",
```
Comment (sampsapenna) [inline swift_browser_ui_frontend/src/common/lang.js:564]: Translation: *Tutkimusprojektini.*

File: swift_browser_ui_frontend/src/views/AddContainer.vue [hunk 3]
```diff
@@ -1,61 +1,72 @@
...
+      <p class="info-text is-size-6">
+        {{ $t("message.container_ops.createdFolder") }}
+        <b>{{ $t("message.container_ops.myResearchProject") }}</b>
+      </p>
+      <c-link
+        :href="`https://my.csc.fi/myProjects/project/${currentProjectID}`"
+        underline
+        target="_blank"
+      >
+        View project members
```
Comment (sampsapenna) [inline swift_browser_ui_frontend/src/views/AddContainer.vue:49]: This is missing internationalisation. Example for Finnish translation: *Näytä projektin jäsenet*

**Review by sampsapenna — CHANGES_REQUESTED** [review]
Added translations for Finnish parts, I can add them myself if necessary. Single line not parameterized. Other than that, LGTM 👍🏼

**Review by sampsapenna — APPROVED** [review]

**Pull request merged**